Makefile

```diff
@@ -107,15 +107,8 @@ SAIL_ARCH_SRCS += riscv_types_kext.sail    # Shared/common code for the cryptogr
 SAIL_STEP_SRCS = riscv_step_common.sail riscv_step_ext.sail riscv_decode_ext.sail riscv_fetch.sail riscv_step.sail
 RVFI_STEP_SRCS = riscv_step_common.sail riscv_step_rvfi.sail riscv_decode_ext.sail riscv_fetch_rvfi.sail riscv_step.sail
 
-# Control inclusion of 64-bit only riscv_analysis
-ifeq ($(ARCH),RV32)
 SAIL_OTHER_SRCS     = $(SAIL_STEP_SRCS)
 SAIL_OTHER_COQ_SRCS = riscv_termination_common.sail riscv_termination_rv32.sail
```

Comment:
This now unconditionally uses riscv_termination_rv32.sail, even for RV64. Are these functions still needed? If so the conditional is still required. I've never used the coq build so not sure about that.

Reply:
That was just an oversight on my part. I actually think they do not need to be separate after Nikhil's refactoring of the virtual memory code.

Reply:
It might also be the case that `SAIL_OTHER_SRCS` is unnecessary now as there is no `riscv_analysis` file, but I'd rather just make the minimal change to the Makefile.

Reply:
Marking as resolved per Alasdair